addons/sale_project/tests/test_sale_project.py

```diff
@@ -426,13 +421,6 @@ def test_compute_project_and_task_button(self):
         sale_order_2._compute_show_project_and_task_button()
         self.assertTrue(sale_order_2.show_create_project_button, "There is a product service with the service_policy set on 'ordered_prepaid' on the sale order, the button should be displayed")
         self.assertFalse(sale_order_2.show_project_button, "There is no project on the sale order, the button should be hidden")
-        self.assertTrue(sale_order_2.show_task_button, "There is no project on the sale order and there is a task whose sale item is one of the sale_line of the SO, the button should be displayed")
-        self.assertEqual(sale_order_2.tasks_ids, task)
-        task.action_convert_to_template()
-        sale_order_2._compute_tasks_ids()
-        sale_order_2._compute_show_project_and_task_button()
-        self.assertFalse(sale_order_2.show_task_button, 'The button should no longer be visible since no tasks are linked to the SO.')
-        self.assertFalse(sale_order_2.tasks_ids, 'No tasks should be linked to the SO since the task has been converted into a template.')
 
```

Comment:
you went a bit too far with your removal here. Couldn't we keep the testing part that test the behavior of the task template conversion ?